### DISCUSSION TITLE
Error in opening with pyMuPDF

### DISCUSSION THREAD

**Discussion by Rafael-da-Rocha** [pymupdf/PyMuPDF#1186] [Looking for help]
I trying open a file with PyMuPDF and receive the message "mupdf: cannot find startxref" and I can open with MuPDF the same file.
how can I fix this?

**Comment by JorjMcKie** [comment]
This message is from MuPDF when it cannot find the XREF table in a damaged PDF.
Of course the same situation also occurs when the MuPDF opens that file, but the message is not displayed.
An automatic repair will happen and the XREF table be reconstructed via a file scan.
Your file should be open / usable in PyMuPDF also. Try `doc.page_count` or similar afterwards.
You can also look for more (warning) messages that may have accompanied that error message: `print(fitz.TOOLS.mupdf_warnings())`.
